---
title: "Short talk: Spiky: standardizing cfMeDIP-seq data with spike-in controls"
draft: false
type: "single"
---

## Spiky: standardizing cfMeDIP-seq data with spike-in controls
Lauren Marie Harmon,Samantha Lea Wilson,Michael Hoffman,Shu Yi Shen,Justin M Burgener,Scott V Bratman,Daniel D. DeCarvalho,Tim Triche
Van Andel Institute
#### Abstract

Cell-free methylated DNA immunoprecipitation-sequencing (cfMeDIP-seq) is a sensitive and template-sparing approach to identify genomic regions with DNA methylation using cell-free DNA. cfMeDIP-seq is well-suited for liquid biopsies due to its low input DNA requirement (1ng) and cost-efficiency. Furthermore, cfMeDIP-seq conserves scarce circulating DNA by avoiding bisulfite conversion, and increases diagnostic yield in independent studies when compared to cfDNA variant analysis. As an enrichment-based protocol, the same strengths that make cfMeDIP attractive for diagnostic use can create challenges in comparing across labs and time points. 
Standardization is important to correct biases due to technical artifacts and variation in enrichment caused by sequence properties (fragment length, GC%, and CpG fraction). We show that synthetic spike-in sequences can effectively reduce technical variance in cfMeDIP-seq data while retaining biologically relevant signal, and present an efficient Bioconductor-compliant package, "spiky," to ease clinical deployment of the flexible standardization workflow. Spiky provides researchers and diagnostic laboratories with  efficient, flexible, and robust methods for the standardization of enrichment-based liquid biopsies, and a foundation for evaluating progressively more sophisticated standardization strategies as the corpus of clinical data accumulates.

**Keywords:** liquid biopsy,standardization,clinical,cell-free,methylome,epigenomics,sequencing
