---
title: "Poster: SVEPT: Structural Variant Effect Prediction Tool. A user-friendly online tool for the interpretation of Structural Variants."
draft: false
type: "single"
---

## SVEPT: Structural Variant Effect Prediction Tool. A user-friendly online tool for the interpretation of Structural Variants.
Víctor Sánchez-Gaya,María Mariner-Faulí,Alvaro Rada-Iglesias
PhD Student
#### Abstract

In the last few years, the role of non-coding regulatory elements and their involvement in human disease have received great attention, motivated by the fact that up to 90% of the disease-associated variants (e.g. SNPs) reside in non-coding sequences. Among the non-coding regulatory sequences, enhancers are particularly important for the proper establishment of cell type-specific gene-expression programs. Furthermore, the disruption of enhancers can lead to human disease (Enhanceropathies) through two main mechanisms: (i) Mutations or or copy number variants can can directly alter the enhancer sequences and thereby affect expression of their target genes; (ii) Structural Variants (SVs) (i.e. deletions, duplications, inversions and translocations) can provoke changes in 3-D chromatin organization that alter neither the enhancers nor their target genes, but rather the physical communication between them. However, most SVs, except those that directly disrupt a gene, remain poorly understood and are frequently annotated as Variants of Uncertain Significance (VUS). In this work, we present SVEPT (Structural Variant Effect Prediction Tool), a software that can aid on the interpretation of Enhanceropathies caused by SVs. SVEPT provides a user-friendly framework that integrates the state of the art knowledge about gene-enhancer regulation with genomic data from different developmental stages to predict the genes potentially affected by SVs. In addition, we used SVEPT to  analyze a cohort of previously reported patients carrying VUS , which researchers can access to identify interesting candidate genes and SVs for downstream experimental validation. The software is accessible through a Shiny app available online.

**Keywords:** Enhancers,Enhanceropathies,Structural Variants,Variants of Uncertain Significance,3-D genome
