---
title: "Package demo: Integrated ATAC-seq Analysis with ATACseqQC / Integrated ChIP-seq Analysis with ChIPpeakAnno"
draft: false
type: "single"
---

## Integrated ATAC-seq Analysis with ATACseqQC / Integrated ChIP-seq Analysis with ChIPpeakAnno
Kai Hu
UMass Medical School
#### Abstract

I am willing to host a workshop for either of the followings: 
# Integrated ATAC-seq Analysis with ATACseqQC
ATAC-seq stands for Assay for Transposase-Accessible Chromatin using sequencing, which leverages a hyperactive transposase to cleave and tag ("tagmentation") double-strand DNA with sequencing adaptors at the same time. ATAC-seq is a faster yet more sensitive method for analyzing genome-wide chromatin accessibility. It is mostly used to map nucleosome positions but can also be applied to investigate transcription factor binding sites and more. This workshop focuses on the downstream analysis (after peak calling) of ATAC-seq data including the diagnostic plotting of fragment size distribution, detection of mitochondria contamination, nucleosome positioning, and TF footprinting with package ATACseqQC.

#1 Integrated ChIP-seq Analysis with ChIPpeakAnno 
(An update to https://github.com/hukai916/IntegratedChIPseqWorkshop)
ChIP-seq combines chromatin immunoprecipitation (ChIP) with next-generation sequencing technology to analyze protein interactions with DNA. It has been widely adopted by biologists to identify transcription factor binding location and to map specific histone modification or nucleosome position on the genome. A typical workflow of ChIP-seq includes experimental design, sequencing, and downstream bioinformatics analysis. This workshop focuses on the bioinformatics part by comparing popularly practiced tools, their advantages, and their limitations. Specifically, we will demonstrate how to perform downstream analysis (after peak calling) with package ChIPpeakAnno.




**Keywords:** ChIPseq,ChIPpeakAnno,Peak Annotation,ATACseq,ATACseqQC
