---
title: "Poster: Assessment of statistical methods from single cell, bulk RNA-seq, and metagenomics applied to microbiome data"
draft: false
type: "single"
---

## Assessment of statistical methods from single cell, bulk RNA-seq, and metagenomics applied to microbiome data
Matteo Calgaro,Chiara Romualdi,Levi Waldron,Davide Risso,Nicola Vitulo
University of Verona
#### Abstract

Background:
The correct identification of differentially abundant microbial taxa between experimental conditions is a methodological and computational challenge. Recent work has produced methods to deal with the high sparsity and compositionality characteristic of microbiome data, but independent benchmarks comparing these to alternatives developed for RNA-seq data analysis are lacking.
Results:
We compare methods developed for single-cell and bulk RNA-seq, and specifically for microbiome data, in terms of suitability of distributional assumptions, ability to control false discoveries, concordance, power, and correct identification of differentially abundant genera. We benchmark these methods using 100 manually curated datasets from 16S and whole metagenome shotgun sequencing.
Conclusions:
The multivariate and compositional methods developed specifically for microbiome analysis did not outperform univariate methods developed for differential expression analysis of RNA-seq data. We recommend a careful exploratory data analysis prior to application of any inferential model and we present a framework to help scientists make an informed choice of analysis methods in a dataset-specific manner.

**Keywords:** Differential abundance,metagenomics,microbiome,benchmark
